.evergreen.yml

```diff
@@ -82,39 +89,35 @@ functions:
         env:
           KUBECONFIG: ${workdir}/kube_config
 
-  run_e2e_test:
-    - command: subprocess.exec
+  black_formatting_test:
+    - command: shell.exec
       type: test
       params:
-        add_to_path:
-          - ${workdir}/bin
         working_dir: mongodb-kubernetes-operator
-        env:
-          KUBECONFIG: ${workdir}/kube_config
-        command: "pip3 install -r ./requirements.txt"
+        script: |
+            . venv/bin/activate
+            pip3 install black
+            black --check `find . -type f -not -path '*venv*' -name '*.py'`
 
-    - command: subprocess.exec
+  run_e2e_test:
+    - command: shell.exec
       type: test
       params:
         working_dir: mongodb-kubernetes-operator
-        add_to_path:
-          - ${workdir}/bin
-        env:
-          KUBECONFIG: ${workdir}/kube_config
-        command: "python3 ./scripts/dev/e2e.py --test ${test} --tag ${version_id} --config_file ./scripts/ci/config.json"
+        script: |
+            . venv/bin/activate
+            pip3 install -r ./requirements.txt
+            KUBECONFIG=${workdir}/kube_config python3 ./scripts/dev/e2e.py --test ${test} --tag ${version_id} --config_file ./scripts/ci/config.json
 
   mypy_linting_test:
-    - command: subprocess.exec
+    - command: shell.exec
       type: test
       params:
         working_dir: mongodb-kubernetes-operator
-        command: "pip3 install mypy"
-
-    - command: subprocess.exec
-      type: test
-      params:
-        working_dir: mongodb-kubernetes-operator
-        command: "find . -type f -name '*.py' -exec python3 -m mypy --ignore-missing-imports {} +"
+        script: |
+            . venv/bin/activate
+            pip3 install mypy
```

Comment:
if the setup of venv already includes mypy, this can turn back into a `command` that just runs mypy right?